src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +81,80 @@ Common questions and requests include:
 
-## Second-shift hero support
+## Second-shift hero support [#emea_support]
 
 Our Europe writers cover the 2nd shift heroing during their regular working hours. Unlike the US-based heroes, our Barcelona writers hero for an entire two-week sprint. Second-shift heroes cover both GitHub and Slack, but we don't expect that second-shift heroes will field every single request that comes in during their working hours since they're also carrying standard sprint duties during their hero shift.
 
-## Route and triage user feedback
-
-The hero is responsible for routing user feedback to liasions. This sections covers the entire lifecycle of a user feedback ticket:
-
-1. The hero is responsible for routing incoming tickets to the proper liaison each day
-    - The Hero leaves a comment saying the ticket is being triaged
-2. Each writer should spend 10-15 minutes/day triaging their assigned tickets 
-    - Writer either rewrites or closes the ticket
-      - If the writer closes the ticket, they must leave a comment explaining why.
-    - Writer needs to add: 
-      - Acceptance criteria
-      - User story
-      - Action items
-      - Resources 
-    - While rewriting, focus on making the ticket as swarmable as possible
-    - Assign a priority [See priority info in our team's `DOC Jira and Github fields - Project 401 - 2022` doc]
-    - Add the following label: `doc_groomed`
-3. Triaged tickets enter sprints in one of two ways:
-    - **Scrum leaders**: Bring in tickets based on marked priority
-    - **Liaisons**: Bring in high value issues
-    - **Squads**: During backlog grooming the team can chat about bringing in tickets to fill out a sprint
-
 <Callout variant="important">
 For writers in Europe:
 
 - Route incoming issues to the liaison as you have time. As writers in Europe are "eternal heroes" it isn’t fair to have you do this every day.
 - You should still triage issues routed to you from your liaisonships once per day.
  </Callout>
 
+## Customer feedback sidekick [#cf_sidekick]
+
+Customer feedback from our docs site creates tickets in Jira. It's the sidekick's responsibility to look at every ticket that comes in (usually about 5-10 per day) and make decisions about each one.
+
+We have three options for these tickets:
+
+1. Close it.
+2. Fix it.
+3. Backlog it.
+
+<Callout variant="important">
+Unlike the hero, we don't take any points out of our sprint for the sidekick work. If you're spending more than 60 minutes per day on customer feedback tickets, that's too much time! Please let your manager know.
+ </Callout>
+
+### Close customer feedback
+
+Most customer feedback that comes in isn't actionable, either because it verges on nonsense or because the feedback is too non-specific to act on.
```

Comment:
I think this is a little too blithe/dismissive of the value. We should emphasize the value of feedback but talk about the operations we create to make it manageble and meaningful

Reply:
Fair